code/generate_setting/generate_setting.R

```diff
@@ -0,0 +1,20 @@
+args = commandArgs(trailingOnly=FALSE)
```

Comment:
`trailingOnly=FALSE`: add space before and after `=` everywhere (applies to several lines below as well)